### PR TITLE
Add strings for HelpCommand and CommandsCommand, remove old ones, a few fixes

### PR DESCRIPTION
**Pull request by napstr**
Needed for https://github.com/Frederikam/FredBoat/pull/124

### CODE REVIEW

File: en_US.properties [hunk 1]
```diff
@@ -191,7 +176,85 @@ userinfoKnownServer=Known Servers\:
...
+commandsUtility=Utility
+commandsModeration=Moderation
+commandsMaintenance=Maintenance
+commandsBotOwner=Bot owner
+commandsMoreHelp=Use `{0}{1} <command>` for more information on a specific command.
+helpDM=Documentation can be found at the bot''s GitHub pages\:\nhttp\://docs.frederikam.com\n\nWant to add FredBoat to your server? If you have Manage Server permissions for your guild, you can invite it here\:\n*This one doesn't play music*\nhttps\://discordapp.com/oauth2/authorize?&client_id=168686772216135681&scope=bot\n\nIf you want to add the music bot, you will want to invite this bot\:\nhttps\://discordapp.com/oauth2/authorize?&client_id=184405253028970496&scope=bot\n\nNeed help or have any ideas for the bot? Perhaps you just want to hang out? Come on over to FredBoat hangout!\nhttps\://discord.gg/0yXhQ9c36F4zsJMG\n\nYou cannot send this bot commands through DM.\nBot created by Fre_d
+helpSent=Documentation has been sent to your DMs!
+helpProperUsage=Proper usage:
+helpCommandOwnerRestricted=#This command is restricted to the owner of the bot.
+helpUnknownCommand={0}{1}: Unknown Command.
```
Comment (Frederikam) [inline en_US.properties]: Is `{0}{1}` really required? What are these?

Comment (napstr) [inline en_US.properties]: those are prefix + command

Comment (Frederikam) [inline en_US.properties]: I don't think those needs to be in the strings then. They can just be prepended.

**Comment by napstr** [comment]
Requested changes applied.

File: en_US.properties [hunk 2]
```diff
@@ -188,73 +188,41 @@ commandsMoreHelp=Use `{0}{1} <command>` for more information on a specific comma
...
+helpShuffleCommand=Toggle shuffle mode for the current queue.
+helpSkipCommand=Skip the current song, the n\'th song in the queue, or all songs from n to m. Please use in moderation.
+helpStopCommand=Stop the player and clear the playlist. Reserved for moderators with Manage Messages permission.
+helpUnpauseCommand=Unpause the player.
+helpVolumeCommand=Changes the volume. Values are 0-150 and 100 is the default. The volume command is deprecated on the public bot.
+helpExportCommand=Export the current queue to a hastebin link, can be later used as a playlist.
+helpGensokyoRadioCommand=Show the current song played on gensokyoradio.net
+helpListCommand=Display a list of the current songs in the playlist.
+helpNowplayingCommand=Display the currently playing song.
+helpForwardCommand=Forward the track by a given amount of time. Example: {0}{1} 2:30
```
Comment (Frederikam) [inline en_US.properties]: These `{0}{1}` will still confuse translators

**Comment by napstr** [comment]
Requested changes applied.

**Comment by Frederikam** [comment]
Sorry, but you still missed a couple `{0}{1}`

**Comment by napstr** [comment]
This is about making a single {0} out of them, not entirely removing the insertions, right?

> commandsMoreHelp=Use `{0}{1} <command>` for more information on a specific command.
> helpCommandsPromotion=Use `{0}{1}` to learn what this bot can do!

**Comment by Frederikam** [comment]
Yes.

**Comment by napstr** [comment]
Done.

**Pull request merged**